.circleci/config.yml

```diff
@@ -145,7 +145,7 @@ workflows:
           type: approval # <<< This key-value pair will set your workflow to a status of "On Hold"
           # On approval of the `hold` job, any successive job that requires the `hold` job will run.
       - integration_tests:
-          context: supertestaccount
+          context: autotestaccount
```

Comment:
I guess the context has been renamed or a new one is being used now for some reason?

Reply:
autotestaccount provider account used to be old 3scale Service based. It has been "promoted" to APIaP (products and Backends), so e2e tests can be run in this env. I use "supertest" for daily use, so I try to avoid conflicts.

Reply:
:+1